netlify/build

```diff
@@ -23,10 +23,8 @@ cp _site/docs/404.html _site/404.html
 
 # Set up htmltest
 
-curl https://htmltest.wjdp.uk | bash
-echo "debug 0"
+curl https://htmltest.wjdp.uk | bash > /dev/null 2>&1
```

Comment:
This will fail if that site is down but you won't find out because you're redirecting STDERR ... that OK?

Reply:
I fixed it in the event that we can't download htmltest.